dae-alright/src/index.js

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import 'bootstrap/dist/css/bootstrap.min.css';
+// import 'bootstrap/js/dropdown.js';
```

Comment:
Try not to leave comments like this when pushing

Reply:
Oh my bad. I forgot to remove it before pushing